server/index.js

```diff
@@ -44,9 +44,12 @@ app.get('/comments/:id', async (req, res) => {
   try {
     const { id } = req.params;
 
-    const comment = await db.getComment(id);
+    let comment;
+    if (Number.isInteger(JSON.parse(id))) {
```

Comment:
I don't think `JSON.parse(id)` is best here, you're paying in computing power for bells and whistles you don't need. I think just casting to `Number(id)` will be faster. See this [gist](https://gist.github.com/gektin/963b0e82960405fd738f1d8909c82cff).

Same comment applies to line 84.

Reply:
Will update in the next PR